### PR TITLE
mentat 0.1.21

### PR DESCRIPTION
**Pull request by PCSwingle**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
@PCSwingle why close this PR?

File: Formula/m/mentat.rb [hunk 1]
```diff
@@ -28,8 +28,8 @@ class Mentat < Formula
   depends_on "six"
 
   resource "aiohttp" do
-    url "https://files.pythonhosted.org/packages/c4/50/a717a133bda2efc27efbf8a65398c925b6d0605213da0db6929627ccb758/aiohttp-3.9.0b0.tar.gz"
-    sha256 "cecc64fd7bae6debdf43437e3c83183c40d4f4d86486946f412c113960598eee"
+    url "https://files.pythonhosted.org/packages/fd/01/f180d31923751fd20185c96938994823f00918ee5ac7b058edc005382406/aiohttp-3.8.6.tar.gz"
+    sha256 "b0cf2a4501bff9330a8a5248b4ce951851e415bdcce9dc158e76cfd55e15085c"
   end
```
Comment (chenrui333) [inline Formula/m/mentat.rb:33]: revert this change would help with py3.12 build

**Comment by PCSwingle** [comment]
@chenrui333 I noticed after making this PR that homebrew was automatically updating the formula when we updated the library on pypi, so I decided to let homebrew do it instead. I hope this PR didn't mess anything up with regard to that. Thank you!
